docs/install_guides/installing-openvino-from-archive-linux.md

```diff
@@ -48,74 +49,84 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
 
 ## Installing OpenVINO Runtime
 
+### <a name="install-openvino"></a>Step 1: Download and Install the OpenVINO Core Components
+
+1. Open a command prompt terminal window. You can use the keyboard shortcut: Ctrl+Alt+T
+
+2. Create the `/opt/intel` folder for OpenVINO and switch to it by using the following commands. If the folder already exists, skip the `mkdir` command.
+
+   ```sh
+   sudo mkdir /opt/intel
+   cd /opt/intel
+   ```
+
+   > **NOTE**: The `/opt/intel` path is the recommended folder path for administrators or root users. If you prefer to install OpenVINO in regular userspace, the recommended path is `/home/<USER>/intel`. You may use a different path if desired.
+
+3. Download the OpenVINO Runtime 2022.2 archive file from the [OpenVINO releases page](https://github.com/openvinotoolkit/openvino/releases/tag/2022.2.0), extract it, and rename the folder to `openvino_2022.2.0.7713` by using the following commands:
+
 @sphinxdirective
 
-.. important::
-   Before you start your journey with installation of the Intel® Distribution of OpenVINO™ toolkit, we encourage you to check the :ref:`code samples <code samples>` in C, C++, Python and :ref:`notebook tutorials <notebook tutorials>`, so you could see all the amazing things that you can achieve with our tool.
+.. tab:: Ubuntu 18.04
+
+   .. code-block:: sh
+   
+      sudo curl -L https://github.com/openvinotoolkit/openvino/releases/download/2022.2.0/l_openvino_toolkit_ubuntu18_2022.2.0.7713.af16ea1d79a_x86_64.tgz --output openvino_2022.2.0.7713.tgz
+      sudo tar -xf openvino_2022.2.0.7713.tgz
+      sudo mv l_openvino_toolkit_ubuntu18_2022.2.0.7713.af16ea1d79a_x86_64 openvino_2022.2.0.7713
+      
+.. tab:: Ubuntu 20.04
+
+   .. code-block:: sh
+   
+      sudo wget https://github.com/openvinotoolkit/openvino/releases/download/2022.2.0/l_openvino_toolkit_ubuntu20_2022.2.0.7713.af16ea1d79a_x86_64.tgz -O openvino_2022.2.0.7713.tgz
```

Comment:
Is curl not installed by default in Ubuntu 20.04?

Reply:
@EdjeElectronics could you help check?

Reply:
That was a mistake on my part! Sorry, I'll fix it 😄

Reply:
Okay, the Ubuntu 20.04 and Red Hat commands were changed to `curl` instead of `wget`.